src/Strategy/RequestResponseStrategy.php

```diff
@@ -27,4 +27,18 @@ public function dispatch($controller, array $vars)
             'return an instance of [Symfony\Component\HttpFoundation\Response]'
         );
     }
+
+    /**
+     * Get Request either from the container or else create it from globals
+     * @return \Symfony\Component\HttpFoundation\Request
+     */
+    protected function getRequest()
+    {
+        if ($this->getContainer()->isRegistered('Symfony\Component\HttpFoundation\Request') ||
+            $this->getContainer()->isInServiceProvider('Symfony\Component\HttpFoundation\Request') ) {
+            return $this->getContainer()->get('Symfony\Component\HttpFoundation\Request');
+        } else {
+            return $this->getContainer()->get('Symfony\Component\HttpFoundation\Request')->createFromGlobals();
```

Comment:
Yeah, should be `return \Symfony\Component\HttpFoundation\Request::createFromGlobals();` or just `return $this->getContainer()->get('Symfony\Component\HttpFoundation\Request');`